sdp.c

```diff
@@ -59,12 +59,20 @@ janus_sdp *janus_sdp_preparse(void *ice_handle, const char *jsep_sdp, char *erro
 					/* Found mid attribute */
 					if(m->type == JANUS_SDP_AUDIO && m->port > 0) {
 						JANUS_LOG(LOG_VERB, "[%"SCNu64"] Audio mid: %s\n", handle->handle_id, a->value);
+						if(strlen(a->value) > 16) {
```

Comment:
Is `a->value` guaranteed to be non-NULL here? I see that you're checking `a->name` but not both as is done e.g. here: https://github.com/meetecho/janus-gateway/pull/1884/files#diff-e26c229e0800bdaf6e4e995fba5d3c4dR3388

Reply:
Good point, but that's actually in master, not in this patch, so will have to be fixed there.

Reply:
Meaning that we're indeed using `a->value` without a check in that new line in the patch, but it was already used in master anyway (e.g., in the log line before, and in the `g_strdup` after that).

Reply:
Should be fixed now :+1: